index.bs

```diff
@@ -718,24 +721,36 @@ There are three builtins:
 * the [=built-in navigating URL attributes list=].
 
 The <dfn>built-in safe default configuration</dfn> is as follows:
-```
-{
-  elements: [ ... ],
-  attributes: [ ... ],
-}
-```
+
+<pre class="include-code">
+path: builtins/safe-default-configuration.json
+highlight: json
+</pre>
 
 The <dfn>built-in safe baseline configuration</dfn> is meant to block only
-script-content, and nothing else. It is as follows:
-```
-{
-  removeElements: [
-    { name: "script", namespace: "http://www.w3.org/1999/xhtml" },
-    { name: "script", namespace: "http://www.w3.org/2000/svg" }
-  ],
-  removeAttributes: [....],
-}
-```
+script-content. It is as follows:
+
+<pre class="include-code">
+path: builtins/safe-baseline-configuration.json
+highlight: json
+</pre>
+
+<div class="note" role="note">
+
+<span class="marker">Note:</span> The [=remove unsafe=] algorithm specifies
+to additionally remove any [=event handler content attributes=], as defined
+in [[HTML]].
+If a [=user agent=] defines extensions to the [[HTML]] spec with additional
+[=event handler content attributes=], it is its responsibility to decide how
+to handle them. Using the current [=event handler content attributes=] list,
+the safe baseline configuration looks effectively like so:
```

Comment:
Nice, this is very important.
nit picking, but is there a higher level of severity than a "note"? Can we do warnings? :D

Reply:
> Nice, this is very important. nit picking, but is there a higher level of severity than a "note"? Can we do warnings? :D

The tool offers [Notes, Issues, Examples, Advisements](https://speced.github.io/bikeshed/#notes-etc)

HTML suports [highly dramatic warnings](https://html.spec.whatwg.org/#typographic-conventions), with a few usages in the spec, e.g. [here](https://html.spec.whatwg.org/#content-type-sniffing).

If we want to use a warning here, I'd just emulate the CSS in the spec directly.

Reply:
Guess it's best to do what's easiest and not worth nit picking too hard. :) thank you!

Reply:
You could add a source comment to upgrade this to `class=warning` when upstreaming.

Reply:
There's also a pre-defined CSS style for "annoying-warning", but that sounds.. like too much. :)

Reply:
I'm happy with this 🥳  thank you!